src/javascript/audio.js

```diff
@@ -66,8 +70,9 @@ export default class Audio {
 
   playBgm(songToExclude) {
     // randomly play background music
-    if (this.player?.playing() && !songToExclude) return;
+    // if (this.player?.playing() && !songToExclude) return;
```

Comment:
Recommend to delete commented-out code, you still have this code in version control history, but commented-out code tends to like to stick around even after moving stuff around because yourself or someone else might be afraid to remove commented-out code.

Reply:
Make sense, removed now.